README.md

```diff
@@ -65,17 +65,47 @@ function oidc_clients() {
 }
 ```
 
-## Development
-Start local site with:
+## Overriding templates
+The pages provided by this plugin are rendered using templates. If you so wish, you can use your own templates instead of this plugins's [default templates](templates).
+
+To do so, you should create an `openid-connect/` directory under your theme, containing only the templates you wish to override. For example, if you wanted to override `authenticate/main.php` and `authenticate/forbidden.php` you would create them under an `openid-connect/` directory in your theme:
```

Comment:
I am confused by the use of the directory `authenticate/`. It is not necessary when overriding the templates? Why do we have this subdirectory in the first place?

Reply:
`authenticate` is the page for which the templates are, i.e. the page with URI `/openid-connect/authenticate/`. So there is one subdirectory per page.

At the moment there is only one page, but there might be other pages in the future, for example a page where you remove previously granted consent. Each page would get its own subdirectory under `templates/`.

Starting with subdirectories since day one, even if there is only one page at the moment, allows us to not break backwards compatibility once (or if) we do introduce other pages. Because users can override these templates, moving them would break backwards compatibility.

Reply:
Some of the paths in the docs were wrong here, I fixed that in https://github.com/Automattic/wp-openid-connect-server/pull/33